### ISSUE TITLE
In game indicator shows Jolteon while PoGoMap not.

### ISSUE THREAD

**Issue by aviram83** [AHAAAAAAA/PokemonGo-Map#2797]
Yesterday I have saw in my in game indicator A Jolteon while I have run the project in the background.

I have account level 21 in the game and a starter account to use in the map.

Is it possible that a low level account won't be able to track pokemons like Jolteon, or it's a bug?

thanks.

**Comment by codyma94** [comment]
I've experienced the same thing today - so far today, the in game radar has said there was an Abra, a Snorlax, and a Scyther near me, none of which the PoGoMap has been able to find. I'm wondering if this is caused by small holes in the search algorithm (although it seems a bit unlikely). I plan on spending a bit more time digging through the code tonight and I'll update if I find anything

**Comment by juliusrucker25** [comment]
Same here. I'm a noob to this stuff, but i was able to follow the instructions when the first version was released. Been updating the versions but it seems like i'm getting less pokemon too. There's always some showing up in my house and the map no longer detects them.

**Comment by TJohnson93** [comment]
Yeah also found some today (notably a Golduck). @aviram83 That's a good point about the different level accounts, I didn't think of that!

**Comment by codyma94** [comment]
@juliusrucker25 one big change that has happened in the past 24 hours is that Niantic has started rate limiting the requests we can make of their API, which might be part of your issue. If you haven't already, you should make sure to run the script with the `-t 1 -sd 5` and see if that improves the results you're getting. Sorry if you've already done that, in which case my comment will be completely useless to you.

**Comment by TJohnson93** [comment]
@codyma94 As with many others I have found `-t 3 -sd 1.1` to be the fastest and the *most* accurate.

**Comment by codyma94** [comment]
@TJohnson93 hadn't seen that before, I'll be using those values from here on out. Thank you!

**Comment by xphongvn** [comment]
Yeah, I wonder if user level affects the search of pokemon? Since higher level user can see more strong and rare pokemon.

I am using a different account for searching because I'm afraid they will ban my main account one day. And my other account was just level 1. 

Any confident answer about this?

**Comment by CT9AJ** [comment]
Anything under under -sd 5 will essentially hardly show any Pokemon on the map anymore for now.

**Comment by aviram83** [comment]
As mention here I was using -t 5 -st 5, have tried lower values since it was slow but I will give it a try.

**Comment by MRokas** [comment]
Using anything above 3 threads, and/or below 5 scan delay is useless now.

**Comment by juliusrucker25** [comment]
So what sd should we be using? Some are saying lower than 5 is slow but i'm seeing 1.1 seems to be the fastest for others?

**Comment by MRokas** [comment]
SD is scan delay. If you use anything below 5, you will get lots of empty responses.

**Comment by aviram83** [comment]
Some more research I have made:
Delete db
set parameters: -t 1 -sd 6 -st 3 -d
run local server in incognito

I knew for fact that a piedgy was located near me, but didn't saw him in the map, when I have enter into the game and walked around he poped.

**Comment by MRokas** [comment]
-sd 6 -st 3 takes ~2 minutes per scan. Just open scanned location and see how slow it is. There isn't anything to improve, all you can do is to adapt to new system.

**Comment by aviram83** [comment]
@MRokas is the last comment for me?

**Comment by MRokas** [comment]
First two are towards you, 3rd and 4th is just general fact.
